test/Constraints/diagnostics.swift

```diff
@@ -173,7 +173,7 @@ func rdar21080030() {
   if s.count() == 0 {} // expected-error {{cannot call value of non-function type 'Int'}}
 }
 
-// <rdar://problem/21248136> QoI: problem with return type inference mis-diagnosed as invalid arguments
+// <rdar://problem/21248136> QoI: problem with return type inference misdiagnosed as invalid arguments
```

Comment:
Please leave the spelling here alone, as you can't rename the record it refers to.